### PR TITLE
Reformatted dataprofiler/dp_logging.py using flake8; changed first docstring in float, text, and order column profiler modules.

### PR DESCRIPTION
**Pull request by jakleh**

### CODE REVIEW

File: dataprofiler/dp_logging.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+"""coding=utf-8."""
```
Comment (JGSweets) [inline dataprofiler/dp_logging.py:1]: should we not be more explicit here?

File: dataprofiler/dp_logging.py [hunk 2]
```diff
@@ -38,7 +40,9 @@ def get_logger():
 
 def set_verbosity(level):
     """
-    Set verbosity level for DataProfiler logger. Must set it to one of
+    Set verbosity level for DataProfiler logger.
+
+    Must set it to one of
```
Comment (taylorfturner) [inline dataprofiler/dp_logging.py:45]:
```suggestion
    Must set it to one of the following values:
```

File: dataprofiler/dp_logging.py [hunk 3]
```diff
@@ -50,7 +54,7 @@ def set_verbosity(level):
 
 def get_child_logger(name):
     """
-    Returns logger for given filepath
+    Return logger for given filepath.
```
Comment (taylorfturner) [inline dataprofiler/dp_logging.py:57]:
```suggestion
    Return logger for the given filepath.
```

File: dataprofiler/profilers/base_column_profilers.py [hunk 4]
```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env python
 """
-coding=utf-8
-
 Profiles the data.
```
Comment (JGSweets) [inline dataprofiler/profilers/base_column_profilers.py:3]: I would suggest these are abstract classes for profiles.

Comment (taylorfturner) [inline dataprofiler/profilers/base_column_profilers.py:3]:
```suggestion
Lowest level column profiler class to be used further up in the inheritance tree of the package.
```

Comment (taylorfturner) [reply dataprofiler/profilers/base_column_profilers.py:3]: see @JGSweets comment on this line too

**Review by JGSweets — APPROVED** [review]

**Pull request merged**